CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-09-04
+
+Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks/release-tracker).
+
+### Announcements
+* The memory limits of liveness-probe container and node-driver-registrar container running in AzureDisk and AzureFile pods on Windows nodes are increased from 100MiB to 150MiB.
+* Pod Disruption Budget (PDB) improvement: 
+  * AKS node drain failures will have better error messages with more details about PDB and unready pods. 
+  * Upgrade will fail sooner when there is a Pod Disruption Budget with max unavailable == 0.
+* Add VMSS tag and node label ebpf-dataplane=cilium when Cilium is enabled
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+  * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+* Kubernetes 1.21 version has been deprecated as of July 31st, 2022. See [documentation](https://docs.microsoft.com/azure/aks/upgrade-cluster?tabs=azure-cli) on how to upgrade your cluster.
+* Some AKS labels have been deprecated with the Kubernetes 1.24 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+* Docker will no longer be supported as a container runtime on Windows after September 1, 2022. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+
+### Release notes
+* Features
```

Comment:
Should we mention:

- ImageCleaner  --enable-image-cleaner flag.

- support cancelling the previous operation. az aks nodepool operation-abort and az aks operation-abort

Reply:
No, the image cleaner will be public preview at the end of this week. Thus it should not be contained in this week's release.

Reply:
Oh cool. I noticed it moved in the AKS roadmap to public preview so thought I was released already.

Reply:
Mentioned.